### PR TITLE
Fix wrong kfserving url

### PR DESCRIPTION
**Pull request by hougangliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2551)

### CODE REVIEW

File: components/kubeflow/kfserving/src/kfservingdeployer.py [hunk 1]
```diff
@@ -182,7 +182,8 @@ def deploy_model(action, model_name, default_model_uri, canary_model_uri, canary
         print('Sample test commands: ')
         print('# Note: If Istio Ingress gateway is not served with LoadBalancer, use $CLUSTER_NODE_IP:31380 as the ISTIO_INGRESS_ENDPOINT')
         print('ISTIO_INGRESS_ENDPOINT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath=\'{.status.loadBalancer.ingress[0].ip}\')')
-        print('curl -X GET -H "Host: ' + url.sub('', model_status['status']['url']) + '" $ISTIO_INGRESS_ENDPOINT')
+        host, path = url.sub('', model_status['status']['url']).split("/", 1)
```
Comment (hougangliu) [inline components/kubeflow/kfserving/src/kfservingdeployer.py]: model_status['status']['url'] is like `http://flowers-sample.kubeflow.example.com/v1/models/flowers-sample` now

Comment (Ark-kun) [reply components/kubeflow/kfserving/src/kfservingdeployer.py]: Maybe add this as s comment.

Comment (hougangliu) [reply components/kubeflow/kfserving/src/kfservingdeployer.py]: agree, done

**Pull request merged**